src/com/connectsdk/service/capability/MediaControl.java

```diff
@@ -133,13 +133,42 @@ else if (transportState.equals("NO_MEDIA_PRESENT")) {
         }
     }
 
+    /**
+     * Get MediaControl implementation
+     * @return MediaControl
```

Comment:
Return values are not explained here, and in a few methods below. Should they be?